logging/logging.go

```diff
@@ -1,25 +1,55 @@
 package logging
 
 import (
+	"github.com/Jaggernaut555/respecbot-v2/types"
 	"log"
 	"os"
+	"strings"
+	"time"
 )
 
 var (
 	logger    *log.Logger
 	errLogger *log.Logger
+
+	// An API instance to send log messages to
+	apiInstance types.API
 )
 
 func init() {
 	logger = log.New(os.Stdout, "", log.Ldate|log.Ltime)
 	errLogger = log.New(os.Stderr, "ERR - ", log.Ldate|log.Ltime)
 }
 
+func SetAPIInstance(instance types.API) {
```

Comment:
This approach can only ever deal with one API type, which won't work in the long run.